### PR TITLE
FASA Reproduction Craft

### PR DESCRIPTION
**Pull request by stratochief66**
Craft files for the basic NASA craft in RSS/RO constructed from FASA parts.

Includes: Mercury Redstone, Mercury Atlas, Atlas Agena, Gemini Titan,
Apollo Saturn V with LEM.

### CODE REVIEW

**Comment by avielmenter** [comment]
Atlas-Agena Targeting Vehicle launch went well without any changes to the .craft file.

Gemini-Titan was really really close, but at fuel exhaustion I was in a 131 x 161km orbit. 

For Mercury-Redstone, the abort sequence wasn't set in action groups. I also noticed that, for some reason, it's rotated 90 degrees away from where rockets usually are by default. This lead to me going south instead of east, because I wasn't paying attention to the numbers on the nav ball. Otherwise, it worked fine and the staging sequence was good.

The Mercury-Atlas had the same small issues as the Mercury-Redstone (no abort sequence and rotated 90 degrees), but I was able to launch it into orbit and then deorbit the mercury pod successfully nonetheless.

The Saturn V is the only one for which I didn't do a full launch test (I examined it in the VAB). I noticed that the LEM staging wasn't set at all. While docking does tend to mess staging order up, if you have it such that the LEM staging is set then it's usually preserved correctly when you detach the LEM from the CSM in lunar orbit. I also noticed that you did not attach floats, or include an abort sequence. Otherwise, it looked fine. The staging sequence for the rocket itself was good.

TL;DR some minor usability nitpicks, but no major issues. All rockets could be launched successfully and accomplish their respective missions as they are in these craft files.
